client/signup/steps/design-picker/index.jsx

```diff
@@ -104,8 +106,9 @@ export default function DesignPickerStep( props ) {
 	const getEventPropsByDesign = ( design ) => ( {
 		theme: design?.stylesheet ?? `pub/${ design?.theme }`,
 		template: design?.template,
-		is_premium: design?.is_premium,
-		is_externally_managed: design?.is_externally_managed,
```

Comment:
Should we remove the `is_premium` and `is_externally_managed` props from the design object, or maybe compute them on the design object so that any consumer of the design object builds these props in the same way?

Reply:
Eventually I think they should be removed, in favour of working directly on unambiguous tiers. In this case (and probably in more) we should keep calculating them in order to ensure that tracks funnels continue working.

There's a list of places we need to update first on https://github.com/Automattic/dotcom-forge/issues/6349